core/src/main/kotlin/fr/rob/core/network/netty/NettyServerHandler.kt

```diff
@@ -31,6 +31,8 @@ abstract class NettyServerHandler(private val nettyServer: NettyServer) : Channe
                 exception.message?.let { nettyServer.logger.error(it) }
             }
 
+            exception.printStackTrace()
```

Comment:
Maybe not in production.
Furthermore, we can add the stacktrace to the log message